ArcGISRuntimeSDKQt_QMLSamples/Maps/MapReferenceScale/MapReferenceScale.qml

```diff
@@ -0,0 +1,253 @@
+        id: referenceScaleRect
+        anchors {
+            margins: 10
+            left: parent.left
+            top: parent.top
+        }
+        height: 150
+        width: 200
+        color: "transparent"
+
+        MouseArea {
+            anchors.fill: parent
+            onClicked: mouse.accepted = true
+            onWheel: wheel.accepted = true
+        }
+
+        Rectangle {
+            anchors.fill: parent
+            width: referenceScaleRect.width
+            height: referenceScaleRect.height
+            color: "#000000"
+            opacity: .75
+            radius: 5
+            border {
+                color: "#4D4D4D"
+                width: 1
+            }
+
+            ColumnLayout {
+                id: referenceScaleLayout
+                anchors {
+                    fill: parent
+                    margins: 5
+                }
+
+                Text {
+                    color: "#ffffff"
+                    text: qsTr("Current Map Scale 1:" + Math.round(mapView.mapScale))
+                    Layout.alignment: Qt.AlignLeft | Qt.AlignBottom
+                    verticalAlignment: Text.AlignBottom
+                    font {
+                        weight: Font.DemiBold
+                        pixelSize: 11
+                    }
+                    clip: true
+                }
+
+                Text {
+                    color: "#ffffff"
+                    text: qsTr("Select a new reference scale")
+                    Layout.alignment: Qt.AlignLeft | Qt.AlignBottom
+                    verticalAlignment: Text.AlignBottom
+                    font {
+                        weight: Font.DemiBold
+                        pixelSize: 11
+                    }
+                    clip: true
+                }
+
+                ComboBox {
+                    id: scales
+                    font {
+                        weight: Font.DemiBold
+                        pixelSize: 11
+                    }
+                    Layout.fillWidth: true
+                    model: ["1:500000","1:250000","1:100000","1:50000"]
+                    clip: true
+                    Component.onCompleted: applyReferenceScaleToMap()
+                    onActivated: applyReferenceScaleToMap()
+                }
+
+                Button {
+                    text: qsTr("Set Map Scale to Reference Scale")
+                    font {
+                        weight: Font.DemiBold
+                        pixelSize: 11
+                        capitalization: Font.MixedCase
+                    }
+                    Layout.fillWidth: true
+                    clip: true
+                    onClicked: mapView.setViewpointScale(referenceScales[scales.currentIndex])
+                }
+            }
+        }
+    }
+
+    Rectangle {
+        id: operationalLayersList
+        anchors {
+            margins: 5
+            right: parent.right
+            top: parent.top
+        }
+        height: 235
```

Comment:
Let's avoid explicit width/heights if possible. If you can try to have it so the order of how heights/widths are populated comes from the child content and bubbles up to the parent item.